### DISCUSSION TITLE
SSO redirect to http instead of https

### DISCUSSION THREAD

**Discussion by xeac** [pimcore/pimcore#17527] [Q&A / Support]
hello guys,

i am using an VM installation of Pimcore in MS Azure. There is an App Gw (LB) infront of Pimcore server. I configured OpenID module for SSO.  
The problem that i have is redirect uri is  "HTTP" instead of "HTTPS" in the header ( find attached ) 
![image](https://github.com/user-attachments/assets/ecbfdb67-0294-4c2a-b9ef-a0831764eb50)

Do you know how to change the redirect uri to "https"?

I am using Ubuntu server with Apache. 

thanks in advance

**Comment by jdreesen** [comment]
https://symfony.com/doc/6.4/deployment/proxies.html

**Reply by xeac** [reply]
hi @jdreesen ,

thanks for your reply. I tried to implement the solution you provided but it seems i get the same result - still http.
Do you have any other ideas?

**Comment by xeac** [comment]
Guys,
how can i configure Pimcore to trust incoming requests from WAF or Application GW?
thnx
